SPECS/sdbus-cpp/sdbus-cpp.spec

```diff
@@ -0,0 +1,71 @@
+Summary:        sdbus-cpp
+Name:           sdbus-cpp
+Version:        1.1.0
+Release:        0%{?dist}
+License:        LGPLv2+
```

Comment:
We should add an entry to the `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`.
A representative PR would be: 
- #2964

Reply:
Fixed